recipes/fastparquet/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: fastparquet
+  version: "0.0.1"
+
+source:
+  fn: fastparquet-0.0.1.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.tar.gz
+  sha256: "c17b46a6f83314434143ac583968b7b0d99f331c50fae13f82e711b64a926e2a"
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas
```

Comment:
Are pandas, numpy, numba and python-snappy build time requirements? It doesn't look like it, but please double check and remove if they aren't.